### PR TITLE
Upgrade protobuf docker to the latest version, add support to build m…

### PR DESCRIPTION
**Pull request by bogdandrutu**
…ore languages

Signed-off-by: Bogdan Drutu <bogdandrutu@gmail.com>

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -15,23 +15,48 @@ endef
 
 # CI build
 .PHONY: ci
-ci: gen-go gen-java gen-python gen-swagger
+ci: gen-cpp gen-csharp gen-go gen-java gen-objc gen-openapi gen-php gen-python gen-ruby
 
-OTEL_DOCKER_PROTOBUF ?= otel/build-protobuf:0.1.0
+OTEL_DOCKER_PROTOBUF ?= otel/build-protobuf:latest
```
Comment (bogdandrutu) [inline makefile:20]: Will update this to 0.2.0 when released in few minutes

**Comment by bogdandrutu** [comment]
When this is approved I will change the settings to not require `gen-swagger` and require `gen-openapi` :) the action step name changed

File: makefile (hunk 1 above)
Comment (bogdandrutu) [reply makefile]: Done.

**Review by tigrannajaryan — APPROVED** [review]
I don't have a good way to validate all these languages manually. Assuming all is good since CI passed.

**Comment by bogdandrutu** [comment]
I generated all locally and check that all expected messages are there

**Review by carlosalberto — APPROVED** [review]

**Comment by carlosalberto** [comment]
(title is too long IMHO, just FYI ;) )

**Pull request merged**